### ISSUE TITLE
Import from OFF the most scanned products

### ISSUE THREAD

**Issue by raphodn** [openfoodfacts/open-prices#103]
### Story

Currently products are added in Open Prices on a per-scan basis, once a price is added.

What about having a list of the top 500 (or more ?) scanned products in Open Food Facts, and importing them in Open Prices. We could then display them in the frontend, and nudge users to add prices for these products, displaying their price count :)

**Comment by raphodn** [comment]
@raphael0202 @teolemon easy to get the top 500 products and import them in the DB ?
I can then work on gamifying a bit the contribution in the app :)

**Comment by raphael0202** [comment]
Yes it's easy, we can even import the full database, it's not that large

**Comment by raphodn** [comment]
awesome ! + the info on the scan count, maybe just 2023 will do

**Comment by raphodn** [comment]
@raphael0202 I think it's a good idea to import the whole database :+1: 
- will allow us to know quickly for a given product, if there are already associated prices or not, and how many
- will also allow for search (though maybe we'll use searchalicious instead ?)

**Comment by raphael0202** [comment]
Done
